controllers/component_dependency_update_controller.go

```diff
@@ -500,7 +514,7 @@ func generateRenovateConfigForNudge(slug string, repositories []renovateReposito
     	enabledManagers: "regex",
 		customManagers: [
 			{
-            	"fileMatch": [".*Dockerfile.*",".*.yaml",".*Containerfile.*"],
+            	"fileMatch": [{{range .FileMatches}},"{{.}}"{{end}}],
```

Comment:
Do we need to escape or sanitize the input `FileMatches` before generating this config? I think so, but what escaping should be done?

Reply:
I don't know if we need to worry about that for now. If you have permission to modify this then you already have permission to run a pipeline, so I don't think there is a security aspect to it, and there are no real valid uses casess where escaping would be required.

Reply:
Maybe we can just JSON-encode the strings? Are JSON strings valid JS strings?